### ISSUE TITLE
API returning due_date.is_computed as string instead of boolean

### ISSUE THREAD

**Issue by baptchv** [mindee/mindee-api-nodejs#303]
### Prerequisites

Put an X between the brackets on this line if you have done all of the following:

* [X] Reproduced the problem or exposed a new need
* [X] Checked the github existing issues

### Steps to Reproduce

1. `Client.parse` with as custom endpoint which creates a ` PredictResponse<GeneratedV1>` 
2. Extract `due_date` : 
```
  [
    'due_date',
    GeneratedObjectField {
    printableValues: [ 'is_computed', 'polygon', 'value' ],
    confidence: 0.99,
    pageId: 0,
    is_computed: 'false',
    polygon: PositionField {
      pageId: undefined,
      boundingBox: [],
      polygon: [Array],
      quadrangle: [],
      rectangle: []
    },
    value: '2024-07-31'
  }
  ]
```

**Expected behavior:**

As per the documentation `is_computed` is a boolean attribute, I expect it to be returned by the API as boolean.

**Actual behavior:**

`is_computed` is returned as string. 

**Reproduces how often:**
Always.

### Versions

v4.17.0

**Comment by sebastianMindee** [comment]
Thank you for raising this. A bit of context: `GeneratedV1` is mostly used as a catch-all for docTI-created APIs, which don't natively support boolean fields.

I will roll out a fix, as this is indeed unintended, but I advise you use either `FinancialDocumentV1` or `InvoiceV4` for the model you were trying to use there.

**Comment by sebastianMindee** [comment]
Fixed in #304, released in v4.17.1
